ibm/resource_ibm_is_vpc.go

```diff
@@ -486,6 +513,22 @@ func vpcCreate(d *schema.ResourceData, meta interface{}, name, apm, rg string, i
 		return fmt.Errorf("Error while creating VPC err %s\n%s", err, response)
 	}
 	d.SetId(*vpc.ID)
+
+	defaultSGName := d.Get(isVPCIDefaultSecurityGroupName).(string)
```

Comment:
For optional filed pls use d.GetOk(XXXX);ok {
}
you can elimate the check of XX !=""